### PR TITLE
Grid system for layout

### PR DESCRIPTION
**Pull request by aqadir-cc**
see commit notes.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -13,3 +13,5 @@
 /tmp/*
 /public/packs
 /node_modules
+/.idea
```
Comment (mckinley) [inline .gitignore]: ...

Comment (aqadir-cc) [inline .gitignore]: Fixed.

**Comment by aqadir-cc** [comment]
@mckinley Yes it's taken from Bootstrap. I have used bootstrap as it requires less learning curve for this short deadline. I agree and we can shift towards Neat as a long-term solution for grids. So I'm sticking to Bootstrap for now.

**Comment by aqadir-cc** [comment]
This one is ready for review and inclusion. @mckinley please merge it after review as other stories are dependent on this one.

File: app/javascript/style/_base.scss [hunk 2]
```diff
@@ -1,4 +1,6 @@
+@import "mixins";
```
Comment (mckinley) [inline app/javascript/style/_base.scss]: As mentioned, I would like to see variables as the first import.

Comment (mckinley) [inline app/javascript/style/_base.scss]: Maybe Im missing something. But like for example, `$grid-columns` is referenced in a mixin, but defined in _varables. Pretty sure variables should be the first import...

Comment (aqadir-cc) [inline app/javascript/style/_base.scss]: Noted.

**Pull request merged**